modules/animation/include/modules/animation/algorithm/animationrange.h

```diff
@@ -38,23 +40,76 @@ namespace inviwo {
 
 namespace animation {
 
+namespace detail {
+template <typename T>
+struct GetTimeHelper {
```

Comment:
Don't think you need the struct here, jus use a regular templated function. then you also get automatic template argument deduction.
The wrapping struct is mostly needed when one wants to do partial specialisation or provide a customisation point.

Reply:
Sweet, thanks!